lib/vdi.py

```diff
@@ -47,6 +47,13 @@ def clone(self):
     def readonly(self) -> bool:
```

Comment:
```suggestion
    def is_readonly(self) -> bool:
```

IMHO This seems more logical when I read it: we're asking about the value of a parameter. I would tend to say that we should have `get_name` rather than `name` on this class too, but that's not a change added by this PR.